src/main/java/pro/sky/Employee/EmployeeControllerNew.java

```diff
@@ -0,0 +1,40 @@
+package pro.sky.Employee;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/departments")
+public class EmployeeControllerNew {
+    private final DepartmentServiseImpl employeeService;
```

Comment:
```suggestion
    private final DepartmentServise employeeService;
```
инъекцию надо делать по интерфейсу